components/bank/Everything.js

```diff
@@ -59,10 +59,10 @@ export default function Everything() {
               }
             >
               Amazing support team
-            </ListItem>
+            </ListItem> */}
             {Object.entries({
-              'Physical check sending & voiding': '',
-              'Online ACH transfers': '',
+              // 'Physical check sending & voiding': '',
+              'Online direct deposit / ACH transfers': '',
               'Generate attendee legal waivers': '',
               'Instant Google Workspace & email addresses': '',
               'Virtual debit cards (with Apple Pay)': '',
```

Comment:
Can we include Google pay here?
```suggestion
               'Virtual debit cards (compatible with Apple Pay & Google Pay)': '',
```